src/roiextractors/extractors/suite2p/suite2psegmentationextractor.py

```diff
@@ -56,21 +64,38 @@ def __init__(
         self.plane_no = plane_no
         self.folder_path = Path(folder_path)
 
-        self.stat = self._load_npy("stat.npy")
-        self._roi_response_raw = self._load_npy("F.npy", mmap_mode="r").T
-        self._roi_response_neuropil = self._load_npy("Fneu.npy", mmap_mode="r").T
-        self._roi_response_deconvolved = self._load_npy("spks.npy", mmap_mode="r").T
-        self.iscell = self._load_npy("iscell.npy", mmap_mode="r")
-        self.ops = self._load_npy("ops.npy").item()
+        def try_load_npy(filename, mmap_mode=None, fn_transform=lambda x: x):
+            """
+            This function allows for incomplete import of files.
+            """
+            try:
+                return fn_transform(self._attempt_load_npy(filename, mmap_mode=mmap_mode))
+            except FileNotFoundError:
+                if allow_incomplete_import:
+                    warnings.warn(f"File {filename} not found.") if warn_missing_files else None
+                    return None
+                else:
+                    raise FileNotFoundError(f"File {filename} not found.")
+
+        self.stat = try_load_npy("stat.npy")
+        self._roi_response_raw = try_load_npy("F.npy", mmap_mode="r", fn_transform=lambda x: x.T)
+        self._roi_response_neuropil = try_load_npy("Fneu.npy", mmap_mode="r", fn_transform=lambda x: x.T)
+        self._roi_response_deconvolved = try_load_npy("spks.npy", mmap_mode="r", fn_transform=lambda x: x.T)
+        self.iscell = try_load_npy("iscell.npy", mmap_mode="r")
+        self.ops = try_load_npy("ops.npy", fn_transform=lambda x: x.item())
```

Comment:
```suggestion
        self.stat = _attempt_load_npy(filename="stat.npy", mmap_mode="r")
        self._roi_response_raw = _attempt_load_npy(filename="F.npy", mmap_mode="r", transpose=True)
        self._roi_response_neuropil = _attempt_load_npy(filename="Fneu.npy", mmap_mode="r", transpose=True)
        self._roi_response_deconvolved = _attempt_load_npy(filename="spks.npy", mmap_mode="r", transpose=True)
        self.iscell = _attempt_load_npy(filename="iscell.npy", mmap_mode="r")
        self.ops = _attempt_load_npy(filename="ops.npy", mmap_mode="r")
        self.ops = self.ops.item() if self.ops is not None else self.ops
```

After the first comment regarding `attempt_load_npy` was accepted (and with extra simplification I just added), it allows us to remove a lot of this